tests/test_chatapproach.py

```diff
@@ -1,37 +1,46 @@
 import json
 
+import pytest
+from openai.types.chat import ChatCompletion
+
 from approaches.chatreadretrieveread import ChatReadRetrieveReadApproach
 
 
-def test_get_search_query():
-    chat_approach = ChatReadRetrieveReadApproach(
-        None, "", "gpt-35-turbo", "gpt-35-turbo", "", "", "", "", "en-us", "lexicon"
+@pytest.fixture
+def chat_approach():
```

Comment:
Oh great, you made it a fixture! I was thinking this should be a fixture, given its constant use. Thanks!